templates/index.html

```diff
@@ -322,13 +328,40 @@ <h3 class="p-stepped-list__title">Launch a VM</h3>
           </li>
         </ol>
         <p>
-          That’s it! Your OpenStack is ready.
+          That’s it! Your micro cloud is ready.
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip--light">
+    <div class="row u-vertically-center">
+      <div class="col-6">
+        <h2>The team behind MicroStack</h2>
+        <p>MicroStack is maintained by the <a class="p-link--external" href="https://ubuntu.com/openstack">OpenStack team at Canonical</a>. We provide OpenStack packages on Ubuntu and have two OpenStack distributions of our own. MicroStack is an on-rails, opinionated OpenStack for the edge, micro clouds and developers. <a class="p-link--external" href="https://ubuntu.com/openstack/features">Charmed OpenStack</a> is a composable enterprise OpenStack for large scale deployments in data centres.</p>
```

Comment:
```suggestion
        <p>MicroStack is maintained by the <a class="p-link--external" href="https://ubuntu.com/openstack">OpenStack team at Canonical</a>. We provide OpenStack packages on Ubuntu and have two OpenStack distributions of our own. MicroStack is an on-rails, opinionated OpenStack for the edge, micro clouds and developers. <a class="p-link--external" href="https://ubuntu.com/openstack/features">Charmed OpenStack</a> is a composable enterprise OpenStack for large scale deployments in data centres. MicroStack was built by the dedicated OpenStack team at Canonical for the developer community. We also make the Charmed OpenStack for multi-cloud scalable clusters. Please contact us if you have any questions.</p>
```

Reply:
Ohhh followed the texts on design here. Fixed!